h5p-bildetema/src/components/Header/Header.module.scss

```diff
@@ -3,6 +3,9 @@
 .header {
   background-color: $burgundy;
   padding: $spacing--32 0;
+  position: sticky;
+  top: 0;
+  z-index: 2;
```

Comment:
Perhaps we should move this to the list of z-indices in `common/abstracts/_variables.scss` to easier keep track of them.

Reply:
Yes!